index.html

```diff
@@ -2183,6 +2183,31 @@ <h5 id="Nature Traits">Nature Traits</h5>
                <td>Nature Level 1
                </td>
             </tr>
+            <tr>
+               <td>Anthropomorphic Beast
+               </td>
+               <td>
+                  While in Beast Shape, you can manipulate objects and weild weapons.  
+                  This still uses your Beast's stats, and you can't weild a weapon which is
+                  disporportionate for the beast (e.g. a Tiny Animal can't wield a Medium or Heavy weapon).
```

Comment:
lol this is wacky but I'm into it. Sword weilding dog? Alllllrighty. 
Although, can a dog use ranged weapons? That might get a bit too wacky. Unless this is more like - oh I turn into a more werewolf type dog thing instead, in which case we might wanna specify that for clarity.